app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -255,20 +272,24 @@ class BrowserFragment : Fragment(), BackHandler, CoroutineScope {
         )
 
         val actionEmitter = ActionBusFactory.get(this).getManagedEmitter(SearchAction::class.java)
-        sessionId?.let { id ->
-            customTabsIntegration.set(
-                feature = CustomTabsIntegration(
-                    requireContext(),
-                    requireComponents.core.sessionManager,
-                    toolbar,
-                    id,
-                    requireActivity(),
-                    onItemTapped = { actionEmitter.onNext(SearchAction.ToolbarMenuItemTapped(it)) }
-                ),
-                owner = this,
-                view = view
-            )
+
+        sessionId?.let { sessionId ->
+            if (sessionManager.findSessionById(sessionId)?.isCustomTabSession() == true) {
```

Comment:
Sorry about the drive-by comment: If the `sessionId` is not null, mean that the session is a Custom Tabs session without needing to re-verify. Did you see this check working for you rather than the `sessionId` check? That sounds like our Custom Tab detection isn't working accurately (which we've seen act oddly depending on the app calling it).

Reply:
When I remove this check, the title view then displays in normal tabs. A normal session will reach this part of `BrowserFragment`, and we need to only set this `customTabIntegration` if it's a custom tab.

Reply:
This sounds like a bug in that case. I've filed https://github.com/mozilla-mobile/fenix/issues/1639 as a follow up.